panel/widgets/button.py

```diff
@@ -156,7 +152,7 @@ def jscallback(self, args: Dict[str, Any] = {}, **callbacks: str) -> Callback:
         return Callback(self, code=callbacks, args=args)
 
 
-class Button(_ClickButton, TooltipMixin):
+class Button(_ButtonBase, _ClickButton, IconMixin, TooltipMixin):
```

Comment:
_ClickButton used to inherit from `_ButtonBase, IconMixin`, but no longer, so we need to subclass these.